internal/service/advancedclustertpf/model_ClusterDescription20240805.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-framework/types"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
-	"go.mongodb.org/atlas-sdk/v20241113001/admin"
+	"go.mongodb.org/atlas-sdk/v20240805005/admin"
```

Comment:
Related to this file, would keep in mind we are simplifying a lot of logic in `NewReplicationSpecsObjType` with regards to how we read replication specs in SDK v2. Right now we rely exclusively on order when there are multiple replication specs. No action to take now but more to keep in mind as we progress to a more stable implementation

Reply:
Thanks. Noted for next Jira issue.